Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/land.py

```diff
@@ -43,6 +43,15 @@ class Land(Command):
     REMOTE = 'origin'
     MIRROR_TIMEOUT = 60
 
+    @classmethod
+    def revert_branch(cls, repository, remote, branch, code):
+        if run(
+            [repository.executable(), 'branch', '-f', branch, 'remotes/{}/{}'.format(remote, branch)],
+            cwd=repository.root_path,
+        ).returncode:
+            return -1
+        return code
```

Comment:
The return code doesn't feel not quite intuitive from the name `revert_branch`. I would expect it turns success or fail if it returns. I understand the idea by passing `return_code_on_revert_success`, but it might be easier if we just  return whether is success.
And do something like `return 1 if cls.revert_branch(repository, cls.REMOTE, target) else -1`
And for line 177, do
```
cls.revert_branch(repository, cls.REMOTE, target)
return -1
```